.github/workflows/pytest_gpu.yml

```diff
@@ -0,0 +1,210 @@
+        os: [ubuntu-latest]
+        python-version: [3.8]
+        include:
+          - python-version: 3.8
+            pytorch-version: 1.13.1
+            torchscript-version: 1.10.2
+            ray-version: 2.0.0
+    env:
+      PYTORCH: ${{ matrix.pytorch-version }}
+      NEUROPOD_BASE_DIR: "/usr/local/lib/neuropod"
+      NEUROPOD_VERSION: "0.3.0-rc6"
+      TORCHSCRIPT_VERSION: ${{ matrix.torchscript-version }}
+      RAY_VERSION: ${{ matrix.ray-version }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.LUDWIG_TESTS_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.LUDWIG_TESTS_AWS_SECRET_ACCESS_KEY }}
+
+    name: py${{ matrix.python-version  }}, torch-${{ matrix.pytorch-version }}, gpu
+
+    timeout-minutes: 90
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Setup Linux
+        if: runner.os == 'linux'
+        run: |
+          sudo apt-get update && sudo apt-get install -y libsndfile1 cmake ccache build-essential g++-8 gcc-8
+          cmake --version
+
+      - name: Install CUDA drivers
+        run: |
+          wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
+          sudo mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
+          wget https://developer.download.nvidia.com/compute/cuda/11.5.1/local_installers/cuda-repo-ubuntu2004-11-5-local_11.5.1-495.29.05-1_amd64.deb
+          sudo dpkg -i cuda-repo-ubuntu2004-11-5-local_11.5.1-495.29.05-1_amd64.deb
+          sudo apt-key add /var/cuda-repo-ubuntu2004-11-5-local/7fa2af80.pub
+          sudo apt-get update
+          sudo apt-get -y install cuda
+        shell: bash
+
+      - name: pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-py${{ matrix.python-version }}-torch${{ matrix.pytorch-version }}-${{ hashFiles('requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-py${{ matrix.python-version }}-torch${{ matrix.pytorch-version }}-
+
+      - name: Install dependencies
+        env:
+          HOROVOD_WITH_PYTORCH: 1
+          HOROVOD_WITHOUT_MPI: 1
+          HOROVOD_WITHOUT_TENSORFLOW: 1
+          HOROVOD_WITHOUT_MXNET: 1
+        run: |
+          python --version
+          pip --version
+          python -m pip install -U pip
+          cmake --version
+
+          echo "MARKERS:" $MARKERS
+
+          if [ "$PYTORCH" == "nightly" ]; then
+            cat requirements.txt | sed '/^torch[>=<]/d' | sed '/^torchtext[>=<]/d' | sed '/^torchvision[>=<]/d' | sed '/^torchaudio[>=<]/d' > requirements-temp && mv requirements-temp requirements.txt
+            pip install --pre torch torchtext torchvision torchaudio
+          else
+            pip install torch==$PYTORCH torchtext torchvision torchaudio
```

Comment:
I believe we need the `--extra-index-url https://download.pytorch.org/whl/cu116`